charts/consul/test/acceptance/tests/basic/basic_test.go

```diff
@@ -1,15 +1,18 @@
 package basic
 
 import (
+	"context"
 	"fmt"
 	"strconv"
+	"strings"
 	"testing"
 
 	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/consul"
 	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/helpers"
 	"github.com/hashicorp/consul-k8s/charts/consul/test/acceptance/framework/logger"
 	"github.com/hashicorp/consul/api"
 	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this is a nit, but can we rename this alias to `metav1`? I don't know if we have been consistent across the project but most kubernetes related imports follow the paradigm `/core/v1` -> `corev1`, `/meta/v1` -> `metav1`

Reply:
Nits are good, I'm still learning what idiomatic code looks like on the team.